File: app/src/main/java/com/musicseque/artist/fragments/MusicFragment.java
package com.musicseque.artist.fragments;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.support.v7.widget.LinearLayoutManager;
import android.support.v7.widget.RecyclerView;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.musicseque.Fonts.Noyhr;
import com.musicseque.R;

import butterknife.BindView;
import butterknife.ButterKnife;

public class MusicFragment extends BaseFragment {
    View view;
    @BindView(R.id.recyclerSongs)
    RecyclerView recyclerSongs;
    @BindView(R.id.tvNoAudio)
    Noyhr tvNoAudio;

    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        view = inflater.inflate(R.layout.fragment_songs, null);
        ButterKnife.bind(this, view);

        initOtherViews();
        initViews();
        listeners();
        return view;
    }


    private void initOtherViews() {

    }

    private void initViews() {
        recyclerSongs.setLayoutManager(new LinearLayoutManager(getActivity(), LinearLayoutManager.VERTICAL, false));

        recyclerSongs.setVisibility(View.GONE);

        tvNoAudio.setVisibility(View.VISIBLE);
        // recyclerSongs.setAdapter(new MusicAdapter(recyclerSongs,getActivity()));
    }


    private void listeners() {

    }
}

File: app/src/main/java/com/musicseque/artist/fragments/MusicFragment.kt
package com.musicseque.artist.fragments

import android.os.Bundle
import android.support.v7.widget.LinearLayoutManager
import android.support.v7.widget.RecyclerView
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import butterknife.BindView
import butterknife.ButterKnife
import com.musicseque.Fonts.Noyhr
import com.musicseque.R
import kotlinx.android.synthetic.main.fragment_songs.*

class MusicFragment : BaseFragment() {
    var v: View? = null


    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        v = inflater.inflate(R.layout.fragment_songs, null)
        initOtherViews()
        initViews()
        listeners()
        return view
    }

    private fun initOtherViews() {}
    private fun initViews() {
        recyclerSongs!!.layoutManager = LinearLayoutManager(activity, LinearLayoutManager.VERTICAL, false)
        recyclerSongs!!.visibility = View.GONE
        tvNoAudio!!.visibility = View.VISIBLE
        // recyclerSongs.setAdapter(new MusicAdapter(recyclerSongs,getActivity()));
    }

    private fun listeners() {}
}